dependencies/Makefile.linux

```diff
@@ -15,11 +15,12 @@ PICO_PACKAGE = libpico.tar.bz2
 LUA_PACKAGE = lua-5.2.3.tar.gz
 OPENSSL_PACKAGE = libssl_1.1.tar.xz
 
-PACKAGES = qt open-al ois pico lua-gd lua assimp open-ssl
+PACKAGES = qt open-al ois pico lua-gd lua assimp
```

Comment:
`lua-gd` target has been removed in another PR but not from the list of targets.
```suggestion
PACKAGES = qt open-al ois pico lua assimp
```

Reply:
Well spotted!